applications/mobu/tests/github_ci_app_enabled_test.yaml

```diff
@@ -52,6 +52,7 @@ tests:
           path: "data['config.yaml']"
           value: |
             autostart: []
+            available_services: []
```

Comment:
Here and elsewhere, this should be `availableServices` to match the Helm case convention (as with `githubCiApp`).